homeassistant/components/devolo_home_network/binary_sensor.py

```diff
@@ -79,12 +81,12 @@ async def async_setup_entry(
     async_add_entities(entities)
 
 
-class DevoloBinarySensorEntity(DevoloEntity, BinarySensorEntity):
+class DevoloBinarySensorEntity(DevoloEntity[LogicalNetwork], BinarySensorEntity):
```

Comment:
Maybe just one small question: there might be additional binary sensors in the future. Should be already introduce a _DataT TypeVar that in bound to LogicalNetwork in preperation here?

Reply:
> Maybe just one small question: there might be additional binary sensors in the future. Should be already introduce a _DataT TypeVar that in bound to LogicalNetwork in preperation here?

We could but honestly, I wouldn't do it now. Better to do when it's actually needed. The template is in place already with `sensor.py`. You should be able to copy / past (most) of that.